### PR TITLE
Quick start removed and self-eval used instead

### PR DESCRIPTION
**Pull request by mdeneva**
Can someone please review to make sure I removed the correct links to Quickstart. I have removed the Quickstart guide and put self-eval in its place

### CODE REVIEW

**Review by cvlc — APPROVED** [review]
Great first step towards consolidating our installation methods, LGTM!

File: docs/operations/troubleshooting.md [hunk 1]
```diff
@@ -240,7 +240,7 @@ StorageClass expects it or that is has been deployed with a different name.
 ### Solution
 
 Deploy Ondat following the [installation procedures](
-/docs/introduction/quickstart). If you are using the manifests
+/docs/introduction/self-eval). If you are using the manifests
```
Comment (cannischan) [inline docs/operations/troubleshooting.md:243]: https://docs.ondat.io/docs/introduction/self-eval is not a valid link - this needs to be https://docs.ondat.io/docs/self-eval/ instead.

**Review by cannischan — DISMISSED** [review]
The link is not valid that's why the linting check is failing - please have a look again

File: docs/operations/troubleshooting.md [hunk 2]
```diff
@@ -240,7 +240,7 @@ StorageClass expects it or that is has been deployed with a different name.
 ### Solution
 
 Deploy Ondat following the [installation procedures](
-/docs/introduction/quickstart). If you are using the manifests
+/docs/introduction/self-eval/). If you are using the manifests
```
Comment (cannischan) [inline docs/operations/troubleshooting.md:243]: Please update this link to /docs/self-eval/

File: docs/operations/licensing.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ kubectl port-forward -n storageos svc/storageos 5705
 The Ondat GUI can then be accessed from `http://localhost:5705` using the default
 `storageos-api` secret defined during the installation as credentials (for
 example: login: storageos / password: storageos) when using the
-[self-evaluation guide](/docs/self-eval/).
```
Comment (cannischan) [inline docs/operations/licensing.md]: this is correct - the new change is incorrect and would go to 404

**Comment by cannischan** [comment]
Apologies I got confused by the linting error - please ignore the link checks before they won't be valid until this PR is merged

**Comment by cvlc** [comment]
I've fixed the last two references to the old self-eval link!

**Comment by cannischan** [comment]
Do not merge until redirect is set up please - thanks!

**Pull request merged**